delta/plugins/elasticsearch/src/main/scala/ch/epfl/bluebrain/nexus/delta/plugins/elasticsearch/ElasticSearchPluginModule.scala

```diff
@@ -325,17 +350,21 @@ class ElasticSearchPluginModule(priority: Int) extends ModuleDef {
         es: ElasticSearchViewsRoutes,
         query: ElasticSearchQueryRoutes,
         indexing: ElasticSearchIndexingRoutes,
+        idResolutionRoute: IdResolutionRoutes,
         schemeDirectives: DeltaSchemeDirectives,
         baseUri: BaseUri
     ) =>
       PriorityRoute(
         priority,
-        ElasticSearchViewsRoutesHandler(
-          schemeDirectives,
-          es.routes,
-          query.routes,
-          indexing.routes
-        )(baseUri),
+        Directives.concat(
+          ElasticSearchViewsRoutesHandler(
```

Comment:
It takes a varargs so you could fit the new route in there.
It should also handle the prefix so you would not have to repeat it in the new route